src/input/index.css

```diff
@@ -1,25 +1,10 @@
-.node > label {
-  cursor: pointer;
-  margin-left: 2px;
+.search {
```

Comment:
This would constitute a breaking change as it alters the HTML and CSS structure. Not this particular one but the PR as a whole. There are a few ways to mitigate it:

 - This feature should be opt-in, which it is by the looks of it. So if the user doesn't opt-in, nothing should change for them. That means it should render the same HTML and CSS structure.
 - The other way is to make this a breaking change, which means the structure is altered permanently and we provide a migration path for old users. 

I'd prefer the former but not if it complicates code structure or adds bloat (to testing, performance, maintenance etc.).

Let me see if I can fiddle with it but if you have considered these and have more insights to share, please let me know.